src/test/java/org/apache/sysml/test/integration/functions/data/RangeTest.java

```diff
@@ -0,0 +1,69 @@
+package org.apache.sysml.test.integration.functions.data;
```

Comment:
It's good you've included license headers in new test script pydml files.
Please also include license header in new RangeTest.java to resolve PR build failure:
https://sparktc.ibmcloud.com/jenkins/job/SystemML-PullRequestBuilder/692/